cli/user/src/test/java/io/pravega/cli/user/SecureUserCommandsTest.java

```diff
@@ -0,0 +1,137 @@
+    @Before
+    @Override
+    public void setUp() throws Exception {
+        tlsEnabled = true;
+        authEnabled = true;
+        super.setUp();
+    }
+
+    @After
+    @Override
+    public void tearDown() throws Exception {
+        super.tearDown();
+    }
+
+    @Test
+    public void testCommands() throws Exception {
+        final String scope = "SecureTestScope";
+        final String streamName = "SecureTestStream";
+        String stream = NameUtils.getScopedStreamName(scope, streamName);
+        final String tableName = "SecureKvtTestTable";
+        final String table = NameUtils.getScopedStreamName(scope, tableName);
+
+        String commandResult = TestUtils.executeCommand("scope create " + scope, config.get());
+        Assert.assertTrue("ScopeCreateCommand failed.", commandResult.contains("created successfully"));
+        Assert.assertNotNull(ScopeCommand.Create.descriptor());
+
+        // List Streams in scope when it is empty.
+        commandResult = TestUtils.executeCommand("stream list " + scope, config.get());
+        Assert.assertFalse("StreamListCommand failed.", commandResult.contains(streamName));
+
+        commandResult = TestUtils.executeCommand("stream create " + stream, config.get());
+        Assert.assertTrue("StreamCreateCommand failed.", commandResult.contains("created successfully"));
+        Assert.assertNotNull(StreamCommand.Create.descriptor());
+
+        // List Streams in scope when we have one.
+        commandResult = TestUtils.executeCommand("stream list " + scope, config.get());
+        Assert.assertTrue("StreamListCommand failed.", commandResult.contains(streamName));
+        Assert.assertNotNull(StreamCommand.List.descriptor());
+
+        commandResult = TestUtils.executeCommand("stream append " + stream + " 100", config.get());
+        Assert.assertTrue("StreamAppendCommand failed.", commandResult.contains("Done"));
+        Assert.assertNotNull(StreamCommand.Append.descriptor());
+
+        commandResult = TestUtils.executeCommand("stream read " + stream + " true 5", config.get());
+        Assert.assertTrue("StreamReadCommand failed.", commandResult.contains("Done"));
+
+        commandResult = TestUtils.executeCommand("stream append " + stream + " key 100", config.get());
+        Assert.assertTrue("StreamAppendCommand failed.", commandResult.contains("Done"));
+        commandResult = TestUtils.executeCommand("stream read " + stream + " 5", config.get());
+        Assert.assertTrue("StreamReadCommand failed.", commandResult.contains("Done"));
+        Assert.assertNotNull(StreamCommand.Read.descriptor());
+
+        commandResult = TestUtils.executeCommand("stream delete " + stream, config.get());
+        Assert.assertTrue("StreamDeleteCommand failed.", commandResult.contains("deleted successfully"));
+        Assert.assertNotNull(StreamCommand.Delete.descriptor());
+
+        // TODO: Test KVT commands in the secure scenario (auth+TLS).
```

Comment:
Are you going to resolve this TODO?

What about the commented out code below?

Reply:
@anirudhkovuru please, review the guidelines to add TODOs in the code: https://github.com/pravega/pravega/wiki/Contributing#todos, which requires to add a reference to the issue the TODO is related to.
@andreipaduroiu  it looks like this test cannot work today due to a bug that ahs been found: https://github.com/pravega/pravega/issues/5374. Until the bug is fix, this test cannot work combining Auth+TLS.

Reply:
I've removed the commented code and mentioned only the issue.